### DISCUSSION TITLE
work for hire/ need someone's helps to realize AI functions

### DISCUSSION THREAD

**Discussion by elvis0920** [mozilla/hubs#5741] [General Questions & Discussion] [answered]
We have already set up a hubs cloud. Within this HC, we would like to have a 3D object to act as a Voice AI Customer Service, which can communicate with users automatically. We have purchases API to a Voice AI database/package, now we would need:
1) connect API to a 3D object in HC and enable the voice AI function;
2) export the conversation out to an assigned email address.
Not sure if these functions can be realized, but if possible, please feel free to reach me. (email: 306445652@qq.com). Would really appreciate your helps and will pay for your time, if necessary. 

FYI: not able to connect to discord, so post a request here. sorry for any inconvenience causes.

**Comment by pattersonbl2** [comment]
@matthewbcool   any way we can post this in the discord for them?

**Reply by elvis0920** [reply]
That'll be awesome! Appreciated.

**Comment by matthewbcool** [comment, marked as answer]
For all work-for-hire posts, we have a dedicated channel in discord. @elvis0920 Welcome to post there, thank you.

https://discord.gg/Jer5t54U
